app/javascript/components/widgets/numbers/panel.jsx

```diff
@@ -63,15 +76,34 @@ class SubscribedEvents extends React.Component {
         <NumberWrapper>
           <div>
             <Count>{githubCommitCount}</Count>
-            <CountLabel color={colors.pink}>{commitText}</CountLabel>
+            <IconTextWrapper>
+              <BlockIcon
+                src={githubCommitImage}
+                width="24"
+                height="24"
+                alt=""
+              />
+              <CountLabel color={colors.pink}>{commitText}</CountLabel>
+            </IconTextWrapper>
           </div>
           <div>
             <Count>{githubPullCount}</Count>
-            <CountLabel color={colors.yellow}>Pull {requestText}</CountLabel>
+            <IconTextWrapper>
+              <BlockIcon src={githubPullImage} width="24" height="24" alt="" />
+              <CountLabel color={colors.yellow}>Pull {requestText}</CountLabel>
```

Comment:
styled-components can accept props in the style definition. Is this generally preferred to adding inline styles like color, width, height.